apps/frontend/src/app/5_pages/ConvertPage/ConvertPage.tsx

```diff
@@ -256,24 +279,81 @@ const ConvertPage: FC = () => {
 
             <div className="w-full flex flex-row justify-between items-center gap-3 mt-3.5">
               <AmountInput
-                value={amount}
+                value={renderDestinationAmount}
                 label={t(commonTranslations.amount)}
                 readOnly
-                placeholder="0"
+                placeholder={t(commonTranslations.na)}
                 className="w-full flex-grow-0 flex-shrink"
                 dataAttribute="convert-to-amount"
               />
               <Select
                 value={destinationToken}
                 onChange={setDestinationToken}
                 options={destinationTokenOptions}
-                labelRenderer={() => getAssetRenderer(destinationToken)}
                 className="min-w-[6.7rem]"
                 dataAttribute="convert-to-asset"
               />
             </div>
           </div>
 
+          <Accordion
```

Comment:
The advanced settings section should be visible only if there's slippage. I know it's not something we can detect now and there's currently no slippage so you can just show it if you have both source and destination tokens.